lib/Serge/Engine/Plugin/parse_yaml.pm

```diff
@@ -26,6 +26,8 @@ sub init {
         email_from     => 'STRING',
         email_to       => 'ARRAY',
         email_subject  => 'STRING',
+
+        yaml_mode      => 'STRING',
```

Comment:
I'd prefer this to be named `yaml_kind` for consistency with https://github.com/evernote/serge/blob/master/lib/Serge/Engine/Plugin/parse_xml.pm#L27